macos/Classes/LiveKitPlugin.swift

```diff
@@ -0,0 +1 @@
+public class LiveKitPlugin: SwiftLiveKitPlugin {}
```

Comment:
Should the ios `LiveKitPlugin.h/m` files also convert to this?

Reply:
for some reason Flutter/iOS needs the ObjC version, i know its ugly (inconsistent) 😅

Reply:
actually I'll take a look if it will work Swift only! thanks 😁